localization/english/wc_scripted_modifiers_l_english.yml

```diff
@@ -0,0 +1,2 @@
+﻿l_english:
+ RELIGION_OBEY_SCOURGE:0 "Obeys the [scourge|E]: $VALUE|=+0$"
```

Comment:
Why not put this file into the modifier folder?

Reply:
> Why not put this file into the modifier folder?

Which one?

Reply:
the only one with _modifier_ in its name... 😅

Reply:
> the only one with _modifier_ in its name... 😅

It's file for modifiers, not scripted modifiers.